code/nfts/nfts-mint-addresses/mint-addresses.en.ts

```diff
@@ -0,0 +1,53 @@
+import { Connection } from '@metaplex/js';
+import { programs } from '@metaplex/js';
+const { metadata: { Metadata, MetadataProgram } } = programs
```

Comment:
In the latest build of `@metaplex/js`, these got moved out to `@metaplex-foundation/mpl-token-metadata`.

The line should be 
```javascript
import { Metadata, MetadataProgram } from '@metaplex-foundation/mpl-token-metadata';
```
instead